java/core/src/test/org/apache/orc/TestReader.java

```diff
@@ -78,4 +78,11 @@ public void testReadFileInvalidHeader() throws Exception {
     OrcFile.createReader(testFilePath,
       OrcFile.readerOptions(conf).filesystem(fs));
   }
+
+  @Test
+  public void testReadDocColumn() throws Exception {
+    Path path = new Path(getClass().getClassLoader().getSystemResource("col.dot.orc").getPath());
```

Comment:
Minor comment: Write, and Read the file within the Test itselft as it would more self-explanatory (col-names, rows etc.)